src/server/project.ts

```diff
@@ -606,6 +614,39 @@ namespace ts.server {
                 }
             }
 
+            if (hasChanges) {

```

Comment:
It was this way before, but I would prefer `const hasChanges = ...;` over `let hasChanges = false; if (...) { hasChanges = true; }